docs/examples/kwok-in-cluster.md

```diff
@@ -0,0 +1,45 @@
+# Deploy Kwok in a Cluster
+
+Define a place to work
+
+``` bash
+KWOK_WORK_DIR=$(mktemp -d)
+```
+
+Set Kwok repository
+
+``` bash
+KWOK_REPO=kubernetes-sigs/kwok
+```
+
+Get latest version
+
+``` bash
+KWOK_LATEST_RELEASE=$(curl "https://api.github.com/repos/${KWOK_REPO}/releases/latest" | jq -r '.tag_name')
+```
+
+Create kustomization file
+
```

Comment:
Firstly, generate a kustomization template yaml to the previously-defined temporary directory.